### PR TITLE
Add Missing "AC, SEER X" to housing characteristics

### PR DESCRIPTION
**Pull request by jmarrec**
## Pull Request Description

Add Missing "AC, SEER X" to housing characteristics TSVs, where X in [14, 18, 24.5]

## Checklist

Not all may apply:

- [ ] Tests (and test files) have been updated
- [ ] Documentation has been updated
- [ ] Changelog has been updated
- [ ] `openstudio tasks.rb update_measures` has been run
- [ ] No unexpected regression test changes on CI (checked comparison artifacts)

### CODE REVIEW

File: project_national/housing_characteristics/HVAC System Single Speed AC Airflow.tsv [hunk 1]
```diff
@@ -13,7 +16,10 @@ Shared Cooling	Yes	0	0	0	0	0	0	0	0	0	0	0	0	1	0
 AC, SEER 8	No	0	0	0	0	0	0	0	0	0	0	0	0	1	0.0134103
 AC, SEER 10	No	0	0	0	0	0	0	0	0	0	0	0	0	1	0.09937
 AC, SEER 13	No	0	0	0	0	0	0	0	0	0	0	0	0	1	0.271354
+AC, SEER 14	No	0	0	0	0	0	0	0	0	0	0	0	0	1	0.124888
 AC, SEER 15	No	0	0	0	0	0	0	0	0	0	0	0	0	1	0.124888
+AC, SEER 18	No	0	0	0	0	0	0	0	0	0	0	0	0	1	0.124888
+AC, SEER 24.5	No	0	0	0	0	0	0	0	0	0	0	0	0	1	0.124888
```
Comment (jason-curtis) [inline project_national/housing_characteristics/HVAC System Single Speed AC Airflow.tsv]: Should last column here be zero so as to not mess up the statistics of the conditional probability table?

File: project_national/housing_characteristics/HVAC System Single Speed AC Airflow.tsv [hunk 2]
```diff
@@ -22,4 +28,4 @@ Room AC, EER 10.7	No	0	0	0	0	0	0	0	0	0	0	0	0	1	0.0941834
 Room AC, EER 12.0	No	0	0	0	0	0	0	0	0	0	0	0	0	1	0.0346726
 Shared Cooling	No	0	0	0	0	0	0	0	0	0	0	0	0	1	0.0185925
 # Created by: manually created
-# Source: Winkler et al. 'Impact of installation faults in air conditioners and heat pumps in single-family homes on US energy usage' 2020
\ No newline at end of file
+# Source: Winkler et al. 'Impact of installation faults in air conditioners and heat pumps in single-family homes on US energy usage' 2020
```
Comment (jason-curtis) [inline project_national/housing_characteristics/HVAC System Single Speed AC Airflow.tsv:31]: probably should update docs here to note that we changed stuff

**Review by jason-curtis — CHANGES_REQUESTED** [review]

File: project_national/housing_characteristics/HVAC System Single Speed AC Airflow.tsv (hunk 1 above)
Comment (jmarrec) [reply project_national/housing_characteristics/HVAC System Single Speed AC Airflow.tsv]: Done.

File: project_national/housing_characteristics/HVAC System Single Speed AC Airflow.tsv (hunk 2 above)
Comment (jmarrec) [reply project_national/housing_characteristics/HVAC System Single Speed AC Airflow.tsv:31]: Done.

**Pull request merged**